eventListener.js

```diff
@@ -0,0 +1,101 @@
+const Web3 = require('web3');
+
+class Web3EventsListener {
+    constructor() {
+        this.activeEvents = {};
+        this.web3Obj = null;
+        this.subscriberId = 0;
+        this.setEvent = this.setEvent.bind(this);
+        this.subscriber = this.subscriber.bind(this);
+        this.subscribeToLogs = this.subscribeToLogs.bind(this);
+        this.subscribeToPendingTransactions = this.subscribeToPendingTransactions.bind(this);
+        this.subscribeToNewBlockHeaders = this.subscribeToNewBlockHeaders.bind(this);
+        this.subscribeToSyncing = this.subscribeToSyncing.bind(this);
+        this.unSubscribe = this.unSubscribe.bind(this);
+    }
+
+    setEvent (type, params, cb, eventEmitter) {
+      if (!this.web3Obj) {
+        const web3Provider = new Web3(new Web3.providers.WebsocketProvider('wss://ropsten.infura.io/ws'));
+        this.web3Obj = new Web3(web3Provider);
+      }
```

Comment:
This doesn't seem correct, you are instantiating `web3Provider` with `new Web3` and then calling it again.

Furthermore, you can do this in one line:
```
this.web3Instance = !this.web3Instance && new Web3(new Web3.providers.WebsocketProvider('wss://ropsten.infura.io/ws'));
```